### ISSUE TITLE
Provide screen reader users a keyboard shortcut to jump between settings tree and settings panel

### ISSUE THREAD

**Issue by ajborka** [microsoft/vscode#92487]
@isidorn: Screen reader users don't have a reliable method of jumping between the settings tree and the settings panel where the settings reside. This makes navigating the settings window difficult because we can lose our place in either the settings panel or the tree.

- VSCode Version: 1.44 (Insider's)
- OS Version: Windows 10 (1909), Ubuntu 20.04

Steps to Reproduce:

1. Start JAWS or NVDA on Windows, or Orca in Linux.
2. Start vscode.
3. Press ctrl+, (comma) for settings.
4. attempt to jump between settings tree and settings panel.

Actual results: There is no reliable method for this. The user must rely on screen reader features to navigate, which may or may not be reliable. In the case of Orca, settings in the settings panel is not accessible with arrow keys unless the user presses TAB from the settings tree first.
Expected results: The screen reader user can press a keyboard shortcut to navigate between settings tree and settings panel. The keyboard shortcut should remember last position and return the user to that location.
Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@ajborka this is great feedback. Thanks for sharing. I agree with this, we should have a keyboard shortcuts to move focus between the settings `Table of Content` s and the actual `Content`.
Assigning to @roblourens since he owns the settings UI. Is it possible to introduce such a command? If you are busy could you provide some code pointers so we open this to the community?

@ajborka as a workaround I suggest you try using F1 > open settings (JSON), as that might be easier to use with a screen reader - give it a try.

**Comment by roblourens** [comment]
I added a command but can anyone suggest a keybinding that makes sense and isn't taken? Or we can leave it for users to pick their own keybinding.

**Comment by isidorn** [comment]
@roblourens I think leaving this up to the users is perfectly fine.
Or for example ctrl + k, ctrl + c
K to start the chord, `c` for contents. Or `t` instead of `c` for table

**Comment by ajborka** [comment]
The normal keyboard shortcut to cycle forward in a situation like this is f6 and the reverse is shift+f6. The best is to leave it for people to define.

**Comment by isidorn** [comment]
@roblourens I could not find this command anywhere. I looked in the command palette (while in the settings UX) for "Focus Settings TOC tree"
I also looked for `settings.focusSettingsTOC` in keyboard shortcuts and I could not find it.
Thus reopening so we can clarify what I am missing. Thanks!

**Comment by ajborka** [comment]
@isidorn  focus settings toc is settings.action.focusTOC. I couldn't find the other one though.

**Comment by isidorn** [comment]
@ajborka thanks! Found it now.
I have verified that command focusses the table fo contents view. @roblourens should we also add that command to the command palette so users can discover it?

**Comment by roblourens** [comment]
Yes

**Comment by roblourens** [comment]
https://github.com/microsoft/vscode/issues/92935
